Setup.py linker flags broken for multiple libs.

I am using system libraries for cjson and libfmt.
```
[ 50%] Building python bindings at python/opm/libopmcommon_python.cpython-39-x86_64-linux-gnu.so
  File "/home/mblatt/src/dune/opm/opm-common/opm-parallel/python/setup.py", line 73
    extra_link_args=['-fopenmp', '/usr/lib/x86_64-linux-gnu/libfmt.so.7.1.3';'/usr/lib/x86_64-linux-gnu/libcjson.so']
    ^
SyntaxError: expression cannot contain assignment, perhaps you meant "=="?
make[2]: *** [CMakeFiles/opmcommon_python.dir/build.make:101: python/opm/libopmcommon_python.cpython-39-x86_64-linux-gnu.so] Fehler 1
make[1]: *** [CMakeFiles/Makefile2:1071: CMakeFiles/opmcommon_python.dir/all] Fehler 2
make: *** [Makefile:160: all] Fehler 2
```

there probably should be  a `,` instead `;`

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -397,6 +397,12 @@ if (OPM_ENABLE_PYTHON)
     list(APPEND opm-common_PYTHON_LINKAGE "'${cjson_LIBRARIES}'")
   endif()
 
+  # opm-common_PYTHON_LINKAGE is used verbatim in a listin python.
+  # Hence items need to be comma separated.
+  if (opm-common_PYTHON_LINKAGE)
+    string(REPLACE ";" "," SETUP_PY_LINKAGE "${opm-common_PYTHON_LINKAGE}")
+  endif()
+
   configure_file (${PROJECT_SOURCE_DIR}/python/setup.py.in ${PROJECT_BINARY_DIR}/python/setup.py)
   file(COPY ${PROJECT_SOURCE_DIR}/python/README.md DESTINATION ${PROJECT_BINARY_DIR}/python)
   execute_process(COMMAND ${Python3_EXECUTABLE} target_name.py

--- a/python/setup.py.in
+++ b/python/setup.py.in
@@ -69,7 +69,7 @@ ext_modules = [
         undef_macros=["NDEBUG"],
         include_dirs=[@SETUP_PY_PYBIND_INCLUDE_DIR@],
         extra_compile_args=['-std=c++17', '-fopenmp', @SETUP_PY_FMT_FLAGS@],
-        extra_link_args=['-fopenmp', @opm-common_PYTHON_LINKAGE@]
+        extra_link_args=['-fopenmp', @SETUP_PY_LINKAGE@]
     )
 ]
 

